### ISSUE TITLE
Screen reader is not announcing the name for 'Close button' under 'Add Action' blade.

### ISSUE THREAD

**Issue by hartra344** [Azure/LogicAppsUX#4040]
### Describe the issue

https://msazure.visualstudio.com/One/_workitems/edit/26532299

User Experience:  
Visually impaired users who are screen reader dependent will face difficulty in understanding the name of the control and They will be confused to user to understand for which name of the control they are navigating. 
Note: User credentials should NOT be included in the bug.
Pre-requisites:
Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed.
Navigate to Azure control present on the right side and select it.
Navigate to LogicApp icon present under workspace section, click on that and select 'Create New Project'.
 Navigate select stateful workflow, then enter the name of the workflow.
Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter.
Navigate to 'Open in current window' and select it.
A dialog will appear, select 'Yes', now the workflow will open in new window.
Repro Steps: 
Open Visual Studio app.
Navigate to any workflow and right click on it, then select 'open designer'.
Navigate to '+' button and select 'Add Action' control.
Navigate all the controls of 'Add Action' blade.
Observe whether Screen reader is not announcing the name for 'Close button' under 'Add Action' blade or not.
Actual Result: 
Screen reader is not announcing the name for 'Close button' under 'Add Action' blade.
Observation: Screen reader Narrator is announcing as 'Button'.

NVDA Behavior: Screen reader NVDA is announcing as 'Button'.
JAWS Behavior: Screen reader JAWS is announcing as 'Button'.
Expected result: 
Screen reader is not announcing the name for 'Close button' under 'Add Action' blade.
Ex: Screen reader should announce as 'Close' button.

Note: Same issue exists for Open designer->search
ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance/previous test pass bug id's: 
N/A 
UIA/DOM details available: 
N/A
MAS Reference: 
[MAS 4.1.2 – Name, Role, Value](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D%7d&file=MAS%204.1.2%20%E2%80%93%20Name%2c%20Role%2c%20Value.docx&action=default&mobileredirect=true)
Reference Links: 
[Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”. 
Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Chrome
